### ISSUE TITLE
make comments on method arguments travel into that methods/constructor’s doc

### ISSUE THREAD

**Issue by rkuhn** [lightbend/genjavadoc#4]
```
class A(/** comment */ val x: Int)
```

**Comment by diversit** [comment]
How to document the constructor?
I have tried several options but generated code always has '// not preceding' before the constructor.

Things I tried:
- 2nd block
/**
 * class comment
*/
/**
 * Constructor comment
 */
case class Test(arg: String)

- before class name
case class /** constructor docs */ Test(arg: String)

- after class name
case class Test /** appears at argument */(arg: String)

**Comment by rkuhn** [comment]
Yes, this is an open issue and I have currently no time to work on it—if you have an idea please share.
